plugins/woocommerce-blocks/tests/e2e/tests/templates/template-user-customization.block_theme.spec.ts

```diff
@@ -47,11 +73,41 @@ const templateUserCustomizationTests = [
 		},
 	},
 	{
-		permalink: '/product/hoodie',
+		visitPage: async ( { page } ) => await page.goto( '/product/hoodie' ),
 		templateName: 'Single Product',
 		templatePath: 'woocommerce/woocommerce//single-product',
 		templateType: 'wp_template',
 	},
+	{
+		visitPage: async ( { frontendUtils } ) =>
+			await frontendUtils.goToCart(),
+		templateName: 'Page: Cart',
+		templatePath: 'woocommerce/woocommerce//page-cart',
+		templateType: 'wp_template',
+	},
+	{
+		visitPage: async ( { frontendUtils } ) => {
+			await frontendUtils.goToShop();
+			await frontendUtils.addToCart();
+			await frontendUtils.goToCheckout();
+		},
+		templateName: 'Page: Checkout',
+		templatePath: 'woocommerce/woocommerce//page-checkout',
+		templateType: 'wp_template',
+	},
+	{
+		visitPage: async ( { frontendUtils, page } ) => {
+			const checkoutPage = new CheckoutPage( { page } );
```

Comment:
I don't know if this is the best way to create the `CheckoutPage()` object, but it was the cleanest way I could find without having to extend the `test` object of all tests. :thinking: